client/src/components/Seat/index.js

```diff
@@ -1,23 +1,36 @@
-import React, { useState } from 'react';
+import React, { useState, useContext } from 'react';
 import './style.scss';
+import { ReservationContext } from '../../context/Reservation';
 
-const Seat = () => {
-  const [selected, setSelected] = useState(false);
+const Seat = ({ columnNr, seatNr }) => {
+  const [selectedSeats, setSelectedSeats, tickets, setTickets] = useContext(
```

Comment:
Check in OrderRoute how the tickets should be passed in to the API and perhaps try to prepare the array accordingly.

Reply:
The parent component should generate tickets, probably.